spec/models/pageflow/revision_spec.rb

```diff
@@ -103,31 +103,48 @@ module Pageflow
         expect(copied_revision.emphasize_chapter_beginning).to eq(true)
       end
 
+      it 'copies storylines to new revision' do
+        revision = create(:revision)
+        storyline = create(:storyline, :revision => revision, :configuration => {'some' => 'value'})
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.